pkg/registry/fetch.go

```diff
@@ -60,10 +67,15 @@ func (f *Fetcher) Fetch(dir, registryImage string) error {
 		handler = handleSource
 	}
 
-	if err := handler(img, dir); err != nil {
+	if err := handler(img, tmpDir); err != nil {
 		return err
 	}
 
+	err = filehelpers.CopyDir(tmpDir, dir)
```

Comment:
Did we observe that this was necessary?

Reply:
No, we don't need it. I meant to remove that, good catch